build/cli/Dockerfile

```diff
@@ -1,16 +1,21 @@
-ARG GO_VERSION='1.18'
+ARG GO_VERSION='1.19'
 ARG ALPINE_VERSION='3.15'
```

Comment:
Latest alpine version is 3.16, might as well bump while we are updating versions?